misc/testerina/modules/testerina-core/src/test/resources/functionmocktest4.pkg/tests/test-sub-program.bal

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Shall we remove this newline?

Reply:
Removed in https://github.com/ballerina-platform/ballerina-lang/pull/13418/commits/8f2497b1b2034b18e20f61a0e73bbc140bafb680